### ISSUE TITLE
Vertical views on mobile have offset bullet (ios)

### ISSUE THREAD

**Issue by moremeyou** [lachholden/obsidian-recipe-view#14]
Love this plugin. 
I second all the enhancement requests. Would also be cool to incorporate with the recipe grabber plugin for automatic formatting :)

Only issue so far is vertical views on mobile have offset bullet. They look fine in landscape.

![IMG_245E4898339A-1](https://github.com/lachholden/obsidian-recipe-view/assets/8971804/2bc6ad92-ba27-43fc-bf4a-2b286111bf37)
![IMG_B81906460EE1-1](https://github.com/lachholden/obsidian-recipe-view/assets/8971804/e1c2e89c-b522-4ea8-b36d-cd831f7dcec9)

**Comment by lachholden** [comment]
Yeah, so this is something I have noticed from time to time and am not quite sure how to fix...

When this has happened to me, if I rotate the screen to landscape and then back they look normal – is this also the case for you? It seems like any "major enough" refresh of the display is enough to fix it.

**Comment by alexlafroscia** [comment]
One thing I noticed, that I'm sure is related: the behavior around what part of the list item is selected on "tap" seems to differ between mobile apps and desktop

Here's the same recipe running in both

| macOS | iPadOS |
| :-- | :-- |
| ![CleanShot 2024-11-09 at 09 53 01@2x](https://github.com/user-attachments/assets/96d45e07-741e-482e-bd0a-c620eabfd37c) | ![IMG_0355](https://github.com/user-attachments/assets/acdf62b6-9ec3-40ce-9a8d-7dfc045e6c8d) |

The iPhone experience is the same as iPadOS, for what it's worth

**Comment by DigiRhys** [comment]
For me, this was tied to the "Tab indent size" in the core Obsidian settings (Editor, very bottom of the Behavior section, should be 3rd-to-last option overall). I moved the slider as far left as it goes (2) and that "fixed" the offset when I'm using recipe view on my phone (Samsung s21 ultra).

I'm also using the AnuPpuccin theme so I had to take an additional step to make sure theme options didn't recreate the issue. For AnuPpuccin that means having Style Settings active, then going to the AnuPpuccin settings > File Editor & Markdown Elements > Lists. With "Enable List Styling" on I also cleared the value for "List Indent".

**Comment by colorfulfool** [comment]
MacOS Obsidian uses Chrome for rendering, while iOS Obsidian uses Safari. Lists created by the plugin have illegal markup. Chrome allows it, while Safari yeets the divs out of the labels.

<img width="677" alt="Image" src="https://github.com/user-attachments/assets/9bd9f28a-a2df-42d6-b558-4eac8a431e6a" />

**Comment by colorfulfool** [comment]
Created a PR that fixes it: https://github.com/lachsh/obsidian-recipe-view/pull/41
